async_lua_ev.c

```diff
@@ -50,6 +57,69 @@ static int async_new(lua_State* L) {
     return 1;
 }
 
+struct async_serialized {
+  ev_async*       async;
+  struct ev_loop* loop;
+};
+
+static int async_sz_send(lua_State *L) {
+    void *udata = lua_touserdata(L, lua_upvalueindex(1));
+    struct async_serialized* data = (struct async_serialized*)udata;
+
+    ev_async_send(data->loop, data->async);
```

Comment:
How can we ensure the lifetime of the `data->async` is still valid when this is called?